### PR TITLE
Add FAQ to the FIPS proxy page

### PR DESCRIPTION
**Pull request by Kaderinho**
### Motivation

https://docs-staging.datadoghq.com/nicolas.guerguadj/add-fips-faq/agent/guide/agent-fips-proxy/?tab=baremetalvm#faq

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/agent-fips-proxy.md [hunk 1]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
 
 If you're still unsure about your issue, you may reach out to the [Datadog support][6].
 
+## FAQ
+
+**1. Does the Datadog Agent and the Datadog FIPS proxy have to be on same host?**
+
+Yes, the FIPS compliance is not retained if the Datadog FIPS proxy and the Datadog Agent are not on the same host.
+The same applies if the `fips.enabled` option in not set to `true` in `datadog.yaml`.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:350]:
```suggestion
**1. Do the Datadog Agent and the Datadog Agent FIPS Proxy have to be on same host?**

Yes, FIPS compliance is not retained if the Datadog Agent FIPS Proxy and the Datadog Agent are not on the same host.
Similarly, FIPS compliance is not retained if the `fips.enabled` option is not set to `true` in `datadog.yaml`.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 2]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**2. Are customers responsible for hardening their host?**
+
+Yes, **Host security and hardening are customer responsibilities.**
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:354]:
```suggestion
Yes, host security and hardening are customer responsibilities.
```
The bold here might make this section a little hard to parse for the reader, and it's a one sentence answer, so I don't think we need the added stress here.

File: content/en/agent/guide/agent-fips-proxy.md [hunk 3]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**3. Are the FIPS proxy images hardened?**
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:356]:
```suggestion
**3. Are the Datadog Agent FIPS Proxy images hardened?**
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 4]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+No, FIPS proxy docker images are not hardened.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:358]:
```suggestion
No, FIPS proxy Docker images are not hardened.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 5]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**4. Is the FIPS proxy supported on every platform?**
+
+No, the FIPS proxy is only avaivable for Linux x86_64. See the [Supported platforms and limitations] section for more details.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:362]: I don't think we need this question. It's clearly covered above.

File: content/en/agent/guide/agent-fips-proxy.md [hunk 6]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**5. Are all incoming and outgoing agent communications FIPS supported?**
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:364]:
```suggestion
**5. Are all incoming and outgoing Agent communications FIPS supported?** 
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 7]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+No, FIPS proxy docker images are not hardened.
+
+**4. Is the FIPS proxy supported on every platform?**
+
+No, the FIPS proxy is only avaivable for Linux x86_64. See the [Supported platforms and limitations] section for more details.
+
+**5. Are all incoming and outgoing agent communications FIPS supported?** 
+
+No, only the communications between the Datadog Agent and the Datadog FIPS proxy support FIPS, if they are on the same host.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:366]:
```suggestion
No, only the communications between the Datadog Agent and the Datadog Agent FIPS Proxy support FIPS, and only if the Agent and the proxy are on the same host.
```

**Review by hestonhoffman — COMMENTED** [review]
Some quick edits

File: content/en/agent/guide/agent-fips-proxy.md (hunk 4 above)
Comment (sgnn7) [reply content/en/agent/guide/agent-fips-proxy.md:358]: I'd maybe reword this slightly to:
```
While the images provided are constructed with security in mind, they have not been evaluated against CIS benchmark recommendations or DISA STIG standards.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 8]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**6. Are all communications between the Cluster Agent and Node Agents FIPS supported?**
+
+No, only the communications between the Cluster Agent and the Datadog FIPS proxy support FIPS.
```
Comment (sgnn7) [inline content/en/agent/guide/agent-fips-proxy.md:370]:
```suggestion
Datadog FIPS proxy only secures communication originating from the Cluster Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Cluster Agent or originating from the Cluster Agent are not covered by this solution.
```

File: content/en/agent/guide/agent-fips-proxy.md (hunk 7 above)
Comment (sgnn7) [reply content/en/agent/guide/agent-fips-proxy.md:366]:
```suggestion
Datadog FIPS proxy only secures communication originating from the Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Agent or originating from the Agent are not covered by this solution.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 9]
```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**4. Is the FIPS proxy supported on every platform?**
+
+No, the FIPS proxy is only avaivable for Linux x86_64. See the [Supported platforms and limitations] section for more details.
+
+**5. Are all incoming and outgoing agent communications FIPS supported?** 
+
+No, only the communications between the Datadog Agent and the Datadog FIPS proxy support FIPS, if they are on the same host.
+
+**6. Are all communications between the Cluster Agent and Node Agents FIPS supported?**
```
Comment (sgnn7) [inline content/en/agent/guide/agent-fips-proxy.md:368]: A few more questions to maybe add:
- `Is FIPS compliance retained if we rebuild or reconfigure the FIPS proxy to fit our deployment or testing needs?`
- `My Agent is sending data fine without following a step in the installation documents. Is this ok?`
- `Are Agent release versions tied to FIPS proxy release versions?

**Review by sgnn7 — COMMENTED** [review]
@Kaderinho Left some comments

File: content/en/agent/guide/agent-fips-proxy.md [hunk 10]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
 
 If you're still unsure about your issue, you may reach out to the [Datadog support][6].
 
+## FAQ
+
+**1. Do the Datadog Agent and the Datadog Agent FIPS Proxy have to be on same host?**
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:347]:
```suggestion
**1. Do the Datadog Agent and the Datadog Agent FIPS Proxy have to be on the same host?**
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 11]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**4. Are all incoming and outgoing Agent communications FIPS supported?**
+
+Datadog FIPS proxy only secures communication originating from the Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Agent or originating from the Agent are not covered by this solution.
+
+**5. Are all communications between the Cluster Agent and Node Agents FIPS supported?**
+
+Datadog FIPS proxy only secures communication originating from the Cluster Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Cluster Agent or originating from the Cluster Agent are not covered by this solution.
+
+**6. Is FIPS compliance retained if we rebuild or reconfigure the Datadog FIPS proxy to fit our deployment or testing needs?**
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:368]:
```suggestion
**6. Is FIPS compliance retained if we rebuild or reconfigure the Datadog FIPS Proxy to fit our deployment or testing needs?**
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 12]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+Even though rebuilding, reconfiguring or modifiying the Datadog FIPS proxy might be a technically working setup, Datadog cannot guarantee FIPS compliance if the Datadog FIPS proxy is not used exactly as explained in the documentation.
+
+**7. My Datadog Agent is correctly sending data without following a step in the installation documents. Is my setup FIPS compliant?**
+
+Datadog cannot guarantee FIPS compliance if the Datadog FIPS proxy is not used exactly as explained in the documentation.
+This notably implies having your Datadog Agent configured to communicate to the FIPS proxy by setting the `fips.enabled` option, as well as having a running Datadog FIPS proxy.
+
+**8. Are Datadog Agent release versions tied to Datadog FIPS proxy release versions?**
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:377]:
```suggestion
**8. Are Datadog Agent release versions tied to Datadog Agent FIPS Proxy release versions?**
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 13]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+While the images provided are constructed with security in mind, they have not been evaluated against CIS benchmark recommendations or DISA STIG standards.
+
+**4. Are all incoming and outgoing Agent communications FIPS supported?**
+
+Datadog FIPS proxy only secures communication originating from the Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Agent or originating from the Agent are not covered by this solution.
+
+**5. Are all communications between the Cluster Agent and Node Agents FIPS supported?**
+
+Datadog FIPS proxy only secures communication originating from the Cluster Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Cluster Agent or originating from the Cluster Agent are not covered by this solution.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:366]:
```suggestion
The Datadog Agent FIPS proxy only secures communication originating from the Cluster Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Cluster Agent or originating from the Cluster Agent are not covered by this solution.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 14]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**6. Is FIPS compliance retained if we rebuild or reconfigure the Datadog FIPS proxy to fit our deployment or testing needs?**
+
+Even though rebuilding, reconfiguring or modifiying the Datadog FIPS proxy might be a technically working setup, Datadog cannot guarantee FIPS compliance if the Datadog FIPS proxy is not used exactly as explained in the documentation.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:370]:
```suggestion
Even though rebuilding, reconfiguring or modifiying the Datadog Agent FIPS Proxy might be a technically working setup, Datadog cannot guarantee FIPS compliance if the Datadog Agent FIPS Proxy is not used exactly as explained in the documentation.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 15]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**7. My Datadog Agent is correctly sending data without following a step in the installation documents. Is my setup FIPS compliant?**
+
+Datadog cannot guarantee FIPS compliance if the Datadog FIPS proxy is not used exactly as explained in the documentation.
+This notably implies having your Datadog Agent configured to communicate to the FIPS proxy by setting the `fips.enabled` option, as well as having a running Datadog FIPS proxy.
+
+**8. Are Datadog Agent release versions tied to Datadog FIPS proxy release versions?**
+
+Datadog FIPS proxy releases are decoupled from Datadog Agent releases. Make sure to use the latest Datadog Agent and Datadog FIPS proxy versions
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:379]:
```suggestion
The Datadog Agent FIPS Proxy releases are decoupled from Datadog Agent releases. Make sure to use the latest Datadog Agent and Datadog Agent FIPS Proxy versions.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 16]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+to have all the available products supported by the Datadog Agent and the Datadog FIPS proxy.
```
Comment (hestonhoffman) [inline content/en/agent/guide/agent-fips-proxy.md:380]:
```suggestion
to have all the available products supported by the Datadog Agent and the Datadog Agent FIPS Proxy.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 17]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+The Datadog Agent FIPS Proxy only secures communication originating from the Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Agent or originating from the Agent are not covered by this solution.
+
+**5. Are all communications between the Cluster Agent and Node Agents FIPS supported?**
+
+The Datadog Agent FIPS Proxy only secures communication originating from the Cluster Agent targeting the Datadog intake API endpoints. This means that other forms of communication terminating at the Cluster Agent or originating from the Cluster Agent are not covered by this solution.
+
+**6. Is FIPS compliance retained if we rebuild or reconfigure the Datadog Agent FIPS Proxy to fit our deployment or testing needs?**
+
+Even though rebuilding, reconfiguring or modifying the Datadog Agent FIPS Proxy might be a technically working setup, Datadog cannot guarantee FIPS compliance if the Datadog Agent FIPS Proxy is not used exactly as explained in the documentation.
```
Comment (sgnn7) [inline content/en/agent/guide/agent-fips-proxy.md:370]:
```suggestion
Even though rebuilding, reconfiguring, or modifying the Datadog Agent FIPS Proxy might be a technically working setup, Datadog cannot guarantee FIPS compliance if the Datadog Agent FIPS Proxy is not used exactly as explained in the documentation.
```

File: content/en/agent/guide/agent-fips-proxy.md [hunk 18]
```diff
@@ -342,10 +342,47 @@ For more information about outbound connections from the Agent, see the [Network
...
+
+**7. My Datadog Agent is correctly sending data even though I didn't follow all of the installation steps listed above. Is my setup FIPS compliant?**
+
+Datadog cannot guarantee FIPS compliance if the Datadog Agent FIPS Proxy is not used exactly as explained in the documentation.
```
Comment (sgnn7) [inline content/en/agent/guide/agent-fips-proxy.md:374]:
```suggestion
Datadog cannot guarantee FIPS compliance if the Datadog Agent FIPS Proxy is not used exactly as documented.
```

**Review by sgnn7 — APPROVED** [review]
